### PR TITLE
Add option for Endpoint Security in Intune

### PR DESCRIPTION
**Pull request by JoelJerkin**
Add option for configuration Attack Surface Reducation via Endpoint Security in Intune instead of Device Configuration Profiles
#ATCP

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md [hunk 1]
```diff
@@ -106,6 +107,18 @@ The following procedures for enabling ASR rules include instructions for how to
 
 4. Select **OK** on the three configuration panes. Then select **Create** if you're creating a new endpoint protection file or **Save** if you're editing an existing one.
 
+**Endpoint Security Policy**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md]:
```suggestion
**Endpoint security policy**
```

File: microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md [hunk 2]
```diff
@@ -106,6 +107,18 @@ The following procedures for enabling ASR rules include instructions for how to
...
+1. Select **Endpoint Security** \> **Attack surface reducation**. Choose an existing Attack surface reducation rule or create a new one. To create a new one, select **Create Policy** and enter information for this profile. For **Profile type**, select **Attach surface reducation rules**. If you've chosen an existing profile, select **Properties** and then select **Settings**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md]:
```suggestion
1. Select **Endpoint Security** \> **Attack surface reduction**. Choose an existing ASR rule or create a new one. To create a new one, select **Create Policy** and enter information for this profile. For **Profile type**, select **Attack surface reduction rules**. If you've chosen an existing profile, select **Properties** and then select **Settings**.
```

File: microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md [hunk 3]
```diff
@@ -106,6 +107,18 @@ The following procedures for enabling ASR rules include instructions for how to
...
+
+2. In the **Configuration settings** pane, select select **Attack Surface Reduction**. Select the desired setting for each ASR rule.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md]:
```suggestion
2. In the **Configuration settings** pane, select **Attack Surface Reduction** and then select the desired setting for each ASR rule.
```

File: microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md [hunk 4]
```diff
@@ -106,6 +107,18 @@ The following procedures for enabling ASR rules include instructions for how to
...
+
+3. Under **List of additional folders that need to be protected**, **List of apps that have access to protected folders**, and **Exclude files and paths from attack surface reduction rules**, enter individual files and folders. You can also select **Import** to import a CSV file that contains files and folders to exclude from ASR rules. Each line in the CSV file should be formatted as follows:
+
+   `C:\folder`, `%ProgramFiles%\folder\file`, `C:\path`
+
+4. Select **Next** on the three configuration panes. Then select **Create** if you're creating a new policy or **Save** if you're editing an existing one.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md]:
```suggestion
4. Select **Next** on the three configuration panes, then select **Create** if you're creating a new policy or **Save** if you're editing an existing policy.
```

File: microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md [hunk 5]
```diff
@@ -106,6 +107,18 @@ The following procedures for enabling ASR rules include instructions for how to
...
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JoelJerkin Please implement the feedback,. Thanks

**Comment by JoelJerkin** [comment]
> @JoelJerkin Please implement the feedback,. Thanks

Done.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@jweston-1 Please merge this PR. Thanks

**Comment by BarbaraSelden** [comment]
#sign-off

**Comment by BarbaraSelden** [comment]
@jweston-1  this now appears out of date. can you guys please resolve this so we can gt it published?

**Comment by yogkumgit** [comment]
@JoelJerkin

**Comment by JoelJerkin** [comment]
@BarbaraSelden @yogkumgit Done!

**Comment by yogkumgit** [comment]
@jweston-1  @BarbaraSelden Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd Will you be able to help us here. Thanks

**Comment by jweston-1** [comment]
Done.

**Pull request merged**